src/FeedbackForm/FeedbackForm.test.js

```diff
@@ -0,0 +1,37 @@
+import { render, screen, waitFor } from "@testing-library/react";
+import FeedbackForm from "./";
+import React from "react";
+import userEvent from "@testing-library/user-event";
+
+describe("MultistepForm", () => {
+  const onSubmit = jest.fn();
+
+  beforeEach(() => {
+    render(<FeedbackForm onSubmit={onSubmit} />);
+  });
+
+  it("onSubmit is called when all fields pass Validation", async () => {
+    const floatingButton = screen.getByRole("button", { name: /add/i });
+    userEvent.click(floatingButton);
+
+    const feedback = await screen.findByTestId("SentimentSatisfiedAltIcon");
```

Comment:
This test should be repeated for all 3 sentiment options to ensure they all work.